projects/igniteui-angular/src/lib/grids/filtering/grid-filtering-cell.component.html

```diff
@@ -2,7 +2,7 @@
         <igx-chips-area [attr.draggable]="false" class="igx-filtering-chips">
             <igx-chip #ghostChip [attr.draggable]="false" (click)="onChipClicked()" (keydown)="onChipKeyDown($event)" [displayDensity]="'cosy'">
                 <igx-icon [attr.draggable]="false" igxPrefix>filter_list</igx-icon>
-                <span [attr.draggable]="false">Filter</span>
+                <span [attr.draggable]="false">{{localService.translate('filter')}}</span>
```

Comment:
`localService` name is misleading. Consider renaming it to `localizationService`.